RecoLocalMuon/CSCRecHitD/test/run_on_raw.py

```diff
@@ -51,12 +51,12 @@
 # LogTrace output goes to cout; all other output to "junk.log"
 
 process.load("FWCore.MessageLogger.MessageLogger_cfi")
-process.MessageLogger.categories.append("CSCRecHit")
-##process.MessageLogger.categories.append("CSCRecoConditions")
 # module label is something like "muonCSCDigis"...
 process.MessageLogger.debugModules = cms.untracked.vstring("*")
-process.MessageLogger.destinations = cms.untracked.vstring("cout","junk")
+process.MessageLogger.cerr.enable = False
+process.MessageLogger.files.junk = dict()
```

Comment:
should junk be a PSet?

Reply:
It could be, but the MessageLogger configuration knows that any names mentioned in `files` must be `untracked.PSet` and therefore just using `dict()` is sufficient to setup the category using the defaults for the `PSet`. This is similar to the case where one doesn't have to give the type when resetting a value as in:
```python
p = PSet( foo = cms.untracked.int32(3))
p.foo = 5
```

Reply:
got it, thank you.